specs/_features/eip6914/fork-choice.md

```diff
@@ -0,0 +1,36 @@
+# EIP-6914 -- Fork Choice
+
+## Table of contents
+
+<!-- TOC -->
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+- [Introduction](#introduction)
+- [Fork choice](#fork-choice)
+  - [Handlers](#handlers)
+    - [`on_reused_index`](#on_reused_index)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+<!-- /TOC -->
+
+## Introduction
+
+This is the modification of the fork choice according to EIP-6914.
+
+## Fork choice
+
+A new handler is added with this upgrade:
+
+- `on_reused_index(store, index)` whenever a validator index `index: ValidatorIndex` is reused
```

Comment:
This is a bit wordy, but I'd like to provide a more specific clarification of what this means.

```suggestion
- `on_reused_index(store, index)` whenever a validator index `index: ValidatorIndex` is reused. That is [`get_index_for_new_validator()`](./beacon-chain.md#get_index_for_new_validator) provides an index due a return value of  `True` from [`is_reusable_validator()`](./beacon-chain.md#is_reusable_validator).
```